Lib/inspect.py

```diff
@@ -441,7 +441,7 @@ def isabstract(object):
                 return True
     return False
 
-def getmembers(object, predicate=None):
+def _getmembers(object, predicate, getter):
     """Return all members of an object as (name, value) pairs sorted by name.
     Optionally, only return members that satisfy a given predicate."""
```

Comment:
if this function is converted to an internal implementation detail with non-optional `predicate` and `getter` args, need to either update the docstring to match the change, or drop the docstring altogether